### PR TITLE
Add doc for docker and Raspberry Pi 4

### PR DESCRIPTION
**Pull request by killerwhile**

### CODE REVIEW

File: Full-node-on-raspberry-pi.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+ssh alephium@alephium
+```
+
+If `alephium` host is unknown, you'll have to search for the IP address of the node, most likely on your router configuration app/page.
+
+And that's it, your Raspberry Pi is running Ubuntu 20.04 with Docker, and is ready to run an Alephium full node.
+
+🚀
+
+[Raspberry pi 4](media/pies.jpeg)
```
Comment (polarker) [inline Full-node-on-raspberry-pi.md]: is it on purpose to not embed this image in the readme with `!`?

File: Full-node-on-raspberry-pi.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+# How to run a docker'ized instance of Alephium full node
+
+This second section is not specific to a Raspberry Pi, but can be generalized to and server/vm/computer with SSH access.
```
Comment (polarker) [inline Full-node-on-raspberry-pi.md]: s/and/any ?

**Review by tdroxler — APPROVED** [review]

**Review by polarker — APPROVED** [review]
Not able to reproduce with a Pi, but it LGTM

File: Full-node-on-raspberry-pi.md (hunk 1 above)
Comment (killerwhile) [reply Full-node-on-raspberry-pi.md]: arf, thanks!

**Pull request merged**